index.html

```diff
@@ -3,10 +3,598 @@
+                <article class="child article1">
+                    <div class="article1-container">
+                        <h1 class="aside-title">GEAR</h1>
+                        <h2 class="article1-text">
+                            <a href="#" class="article1-link">The next iPhone may include a Samsung periscope lens</a>
+                        </h2>
+                            <div class="child-one-details">
+                                <p>by Callum Booth <i class="far fa-clock"></i></p>
+                                <p>3 hour ago <i class="fas fa-share-alt"></i>16</p>
+                            </div>
+                    </div>
+                </article>
+                <article class="child article2">
+                    <div class="article1-container">
+                        <h1 class="aside-title">TECH</h1>
+                        <h2 class="article1-text">
+                            <a href="#" class="article1-link">Maserati commits to going all-electric</a>
+                        </h2>
+                            <div class="child-one-details">
+                                <p>by The Conversation <i class="far fa-clock"></i></p>
+                                <p>3 hour ago <i class="fas fa-share-alt"></i>4</p>
+                            </div>
+                    </div>
+                </article>
+            </aside>
+        </section>
+        <section class="main-second">
+            <h1 class="latest-new">Latest news <i class="fas fa-chevron-right"></i></h1>
+            <section class="second-section">
+                <div class="second-catigory">
+                    <div class="first-category">
+                        <img src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/11/The-Official-Cyber-Monday-Mac-Bundle-Ft.-Parallels-Pro-Luminar-4-540x270.jpg" alt="" class="my-image">
+                        <div>
+                            <a href="#" class="second-category-link">Run Windows on your Mac with this Cyber Monday app bundle feat Parallels</a>
+
+                            <a href="#" class="details">TNW Deals . 57 minutes ago</a>
+                        </div>
+                    </div>
+
+                    <div class="first-category">
+                        <img src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/11/TicWatch-Cyber-Monday-540x270.jpeg" alt="" class="my-image">
+
+                        <div>
+                            <a href="#" class="second-category-link">Google smartwatches, drones, and more Cyber Monday deals to check out</a>
+
+                            <a href="#" class="details">TNW Deals . 1 hour ago</a>
+                        </div>
+                    </div>
+
+                    <div class="first-category">
+                        <img src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/11/header-image-apple-samsung-periscope-lens-516x270.png" alt="" class="my-image">
```

Comment:
Always add the `alt` attribute value for all images. Kindly check through the whole file and make the changes.